k8srm-prototype/testdata/pod-two-containers-two-gpus-1.yaml

```diff
@@ -0,0 +1,38 @@
+# A Pod asking for a two Foozer GPUs of the same model, and associating each
+# with a different container.
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: foozer
+  namespace: default
+spec:
+  containers:
+  - image: registry.k8s.io/pause:3.6
+    name: my-container
+    resources:
+      requests:
+        cpu: 10m
+        memory: 10Mi
+    devices:
+    - name: foozer-gpu
+  - image: registry.k8s.io/pause:3.6
+    name: my-other-container
+    resources:
+      requests:
+        cpu: 10m
+        memory: 10Mi
+    devices:
+    - name: other-foozer-gpu
+  deviceClaims:
+  ## PROBLEM: We want to specify matchAttributes here but cannot! In the "unrolled"
```

Comment:
We probably need a second name, inside the claim to reference the "sub claim". Then, the container mount point has to list both names, the main claim and the subclaim.

OR - we only allow one claim in the podspec, and nest all the subclaims in that. Then we only need one claim, it just becomes the entire former "pod spec" piece of this.